mobile/components/artist_app_banner/index.coffee

```diff
@@ -0,0 +1,25 @@
+AppBanner = require '../app_banner/app_banner.coffee'
+{ USER_AGENT, modal } = require('sharify').data
+
+class ArtistAppBanner extends AppBanner
+    template: ->
+      $ require('./index.jade') { modal: modal }
+    
+    @shouldDisplay: ->
```

Comment:
I believe `@` is a reference to `this` in coffeescript and is implicit. I could be wrong but the `@` may not be needed here. #SuperMinor